### ISSUE TITLE
Add rejit to pip

### ISSUE THREAD

**Issue by ziowk** [ziowk/rejit#88]
Not that anyone is going to use it, but it is fun to have a package in pip.

Check how to add a package to pip the right way.

**Comment by jayvdb** [comment]
Yes please!  There are times when this would be a good alternative to `re`, or at least a good partial fit for people who have the ability to add the missing features they need.
